### ISSUE TITLE
Periodical refresh of the OTP placeholder in details view

### ISSUE THREAD

**Issue by Slummi** [strongbox-password-safe/Strongbox#455]
I was very surprised that you already support the newly introduced TOTP placeholder and custom fields of KeePass 2.47.
Good job!

Perhaps it would be good to periodically refresh the placeholder when you are in the entries details view.
If you use the OTP placeholder in the password field (good for sites that allow to append the OTP to the password, like Amazon), it will only be valid for a short time directly after opening the details view, because the field is not refreshed when the code changes.

If you copy the password field from the browse view it's ok, because the placeholder is replaced during copy process.

**Comment by strongbox-mark** [comment]
Interesting. You mean visually only right? If you hit tap the password to copy it in the details view it will copy the latest totp, right?

**Comment by Slummi** [comment]
You‘re right. It‘s only a visual issue. Copying from the details view also uses the latest code. So it‘s not that urgent, because it’s only relevant if you type out a field that uses the placeholder.

**Comment by strongbox-mark** [comment]
OK thanks for confirming @Slummi - Would be nice to fix but fairly cosmetic
